person.rb

```diff
@@ -1,14 +1,15 @@
 require_relative 'nameable'
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] to make this association work Getters and Setters for @rentals should be present.